app/views.py

```diff
@@ -490,129 +525,162 @@ def handle_background_image(user_id):
-   	    raise BadThings('Cannot follow yourself', status_code = 400)
+            raise BadThings('Cannot follow yourself', status_code=400)
         u = g.user.follow_expert(user)
         if u is None:
-            return BadThings('Cannot follow expert %d' % user_id, status_code = 400)
+            return BadThings('Cannot follow expert %d' % user_id, status_code=400)
         db.session.add(u)
         db.session.commit()
         return make_response(jsonify({'message': 'following expert %d successfully' % user_id}), 200)
     elif request.method == 'GET':
-	if g.user.is_following_expert(user):
-	    msg = 'Following target expert %d' % user_id 
-	else:
-	    msg = 'Not following target expert %d' % user_id
+        if g.user.is_following_expert(user):
+            msg = 'Following target expert %d' % user_id
+        else:
+            msg = 'Not following target expert %d' % user_id
         return make_response(jsonify({'message': msg}), 200)
     elif request.method == 'DELETE':
-	#delete the interested-in relationship, i.e., the customer unfollows the expert
-	u = g.user.unfollow_expert(user)
+        # delete the interested-in relationship, i.e., the customer unfollows the expert
+        u = g.user.unfollow_expert(user)
         if u is None:
-            return BadThings('Cannot unfollow expert %d' % user_id, status_code = 400)
+            return BadThings('Cannot unfollow expert %d' % user_id, status_code=400)
         db.session.add(u)
         db.session.commit()
         return make_response(jsonify({'message': 'unfollowing expert %d successfully' % user_id}), 200)
     else:
         return make_response(jsonify({'message': 'ACTION not supported'}), 403)
 
-@app.route('/hometimeline/<int:user_id>', methods = ['GET'])
+
+@app.route('/hometimeline/<int:user_id>', methods=['GET'])
 def get_hometimeline(user_id):
-    user = Expert.query.filter_by(user_id = user_id).first()
+    user = Expert.query.filter_by(user_id=user_id).first()
     home_timeline = TimelineService.load_hometimeline()
-    return jsonify({'home_timeline' : home_timeline})
+    return jsonify({'home_timeline': home_timeline})
+
 
 @app.route("/login/<user>/<password>")
 def test_login(user, password):
     return "user : %s password : %s" % (user, password)
-	
+
+
 @app.route('/test')
 def test():
     print session['categories']
     return "Testing!"
 
+
 """
     Create some data in the db for testing purposes.
     In the command line, run:
     curl -d "" "http://localhost:5000/api/v1/internal/data/"
 """
-@app.route('/api/v1/internal/data/', methods = ['POST'])
+
+
+@app.route('/api/v1/internal/data/', methods=['POST'])
 def create_data():
     create_data_in_db()
     return "Data created!"
 
+
 @app.errorhandler(BadThings)
 def handle_missing_resource(error):
     response = jsonify(error.to_dict())
     response.status_code = error.status_code
     return response
 
+
+@app.route('/api/v1/easemob-test/user', methods=['POST'])
```

Comment:
Yep, the endpoints start with `easemob-test` is just to tell the usage of easemob module. In fact, these endpoints shouldn't be exposed to user, as easemob user registration should be embedded in our own registration procedure.

Should I move these endpoints to a standalone file like 'test.py'?